### ISSUE TITLE
regex-dna to regex-redux because string sub used instead of regex

### ISSUE THREAD

**Issue by igouy** [kostya/jit-benchmarks#13]
_fwiw_ The regex-dna programs were re-written as [regex-redux](https://benchmarksgame-team.pages.debian.net/benchmarksgame/description/regexredux.html#regexredux) because several contributed programs were using string substitution instead of pattern matching.

**Comment by kostya** [comment]
in this benchmark is there different programs? may be i ll change to it.

**Comment by igouy** [comment]
Yes, the regex-dna programs were re-written to do [regex-redux](https://salsa.debian.org/benchmarksgame-team/archive-alioth-benchmarksgame/-/blob/master/contributed-source-code/benchmarksgame/README)

```
regexdna		obsolete,removed
regexredux		ACTIVE
```
